tools/packager/packager.py.in

```diff
@@ -67,12 +69,14 @@ def run_packager():
     parser.add_argument("remain_args", nargs=argparse.REMAINDER)
     args = parser.parse_args(sys.argv[1:])
     cmd_description = "{0} {1}".format(PACKAGER_EXEC, args.cmd)
-    subparser = argparse.ArgumentParser(description=cmd_description)
+    subparser = argparse.ArgumentParser()
```

Comment:
Why remove description?

Reply:
The description was being printed twice (it's included in the usage message).

Reply:
👍